pkg/cluster/fixmcsuserdata.go

```diff
@@ -121,15 +132,15 @@ func (m *manager) fixMCSUserData(ctx context.Context) error {
 			var userData *userData
 			err = codec.NewDecoderBytes(s.Data["userData"], &h).Decode(&userData)
 			if err != nil {
-				return err
+				return fmt.Errorf("failed decoding secret userData: %w", err)
 			}
 
 			var changed bool
 			for i, a := range userData.Ignition.Config.Merge {
 				var _changed bool
 				a.Source, _changed, err = m.fixSource(a.Source)
 				if err != nil {
-					return err
+					return fmt.Errorf("failed fixing domain source: %w", err)
```

Comment:
```suggestion
					return fmt.Errorf("Failed fixing domain source when merging fields: %w", err)
```

Reply:
Errors are meant to have a lower case first letter, not entirely sure why, but it's a lint error otherwise :(